source/plugin/permissions.rst

```diff
@@ -2,9 +2,293 @@
 Permissions
 ===========
 
+Permission
+==========
+
+A permission is a case-insensitive hierarchical string key that is used to determine whether a Subject can do a specific
+action or not.
+The string is split into separate parts using the dot character.
+Permissions should be structured like this.
+
+.. code-block:: text
+    
+    <PluginID>.<MainGroup>.<Subgroup1>...
+
+Allowed characters are:
+
+* "A" - "Z"
+* "a" - "z"
+* "0" - "9"
+* "_"
+* "-"
+* "."
+
+Inheritance
+~~~~~~~~~~~
+
+If a user has the permission ``myPlugin.commands`` then they automatically have all sub permissions such as
+``myPlugin.commands.teleport`` unless the permissions are explicitly removed.
 
 .. note::
 
-    This page has not been filled yet by our Documentation Team. If you feel like you can help, you can do so on `our
-    GitHub repository <https://github.com/spongepowered/spongedocs>`_. Also see the `related GitHub Issue
-    <https://github.com/SpongePowered/SpongeDocs/issues/32>`_ for more information on what is required.
+    There is no such thing as a ``myPlugin.commands.*`` wildcard permission.
+    Use ``myPlugin.commands`` for that.
+
+Structure-Example
+~~~~~~~~~~~~~~~~~
+
+The following example show one possible way of structuring permissions, but following this structure is not required at
+all.
+
+* ``myPlugin``
+    * Grants **full** access to all plugin's permissions.
+
+* ``myPlugin.commands``
+    * Grants **full** access to all plugin's commands.
+
+* ``myPlugin.commands.teleport.execute``
+    * Only grants the user the permission to execute the command. Without this permission he is not able to execute the
+      command even if he has other `teleport` permissions.
+      (With this permission alone the user would only be able to teleport himself to others in his current world.)
+
+* ``myPlugin.commands.teleport.all``
+    * Only grants the user the permission to teleport all players at once.
+
+* ``myPlugin.commands.teleport.worlds``
+    * Only grants the user the permission to teleport to all worlds.
+
+* ``myPlugin.commands.teleport.worlds.mynetherworld``
+    * Only grants the user the permission to teleport to *mynetherworld*.
+
+PermissionDescription
+=====================
+
+The permission description is an utility that is meant to provide the server owner with details on a certain permission.
+PermissionDescriptions are an optional feature a PermissionService can provide. The creation of a PermissionDescription
+does **not** have any impact on whether a permission exists, who has access or what its default value is.
```

Comment:
Change does to should?

Sure a permissions implementation isn't expected to act based on the permission descriptions currently, but they could, and I don't think it would be outside the spec.